dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -720,21 +721,17 @@ def test_categorical_diff(self):
                 "categories": [[], ["y", "n"], ["maybe"]],
                 "gini_impurity": -0.16326530612244894,
                 "unalikeability": -0.19047619047619047,
-                "categorical_count": {"y": 1, "n": 1, "maybe": [None, 2]},
+                "categorical_count": {"y": 1, "n": 1, "maybe": -2},
                 "chi2-test": {
                     "chi2-statistic": 82 / 35,
                     "df": 2,
                     "p-value": 0.3099238764710244,
                 },
+                "psi": 0.0990210257942779,
```

Comment:
PSI value for when new category is added

Reply:
new `pd.Series(["y", "maybe", "y", "y", "n", "n", "maybe"])`
old `df_categorical = pd.Series(["y", "y", "y", "y", "n", "n", "n"])`